tutorials/plugins/editor/import_plugins.rst

```diff
@@ -226,7 +226,7 @@ you do this you have to be careful when you add more presets.
 
 ::
 
-    func _get_import_options(preset):
+    func _get_import_options(preset, index):
         match preset:
```

Comment:
The match statement needs to use the index parameter, the first parameter is the file path.
It would also be good to rename the parameters to match the function definition. (path, preset_index)
The renaming could also be done for the above _get_preset_name function with "preset_index".